styles/styles.css

```
@@ -0,0 +1,543 @@
+@font-face {
+  font-family: "Raleway";
+  font-weight: 800;
+  font-display: swap;
+  font-style: normal;
+  src: url("fonts/raleway/raleway-800") format("woff2");
```

Относительный путь к файлу написан неправильно. Должна быть такая запись:
`src: url("../fonts/raleway/raleway-800.woff2") format("woff2");`

Ваша запись (красным цветом) и как должно быть (зелёным):
```suggestion
  src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
```

Для остальных шрифтов исправьте по аналогии.